.github/workflows/test.yml

```diff
@@ -32,6 +32,8 @@ jobs:
           args: --verbose
       - name: yamllint-lint
         run: yamllint .
+      - name: shellcheck
+        run: shellcheck $(find . -name '*.sh')
```

Comment:
We could use `make lint-shell` - and also for yamllint above `make lint-yaml`.

The point is to keep the same behavior on the CI, and on people laptops.

LMK what you think.

Thanks!

Reply:
LGTM otherwise - and we do need this on the CI for sure.

cc @AkihiroSuda

Reply:
I would prefer CI to be the single source of truth and commands instead of `make` with its tabs vs spaces, but such tooling is not there yet. Fixed.

Reply:
@abitrolly have a look at https://dagger.io/ - clearly CI is a hot steaming pile of mess right now and we definitely need better - dagger is onto that mission.